pilot/pkg/features/pilot.go

```diff
@@ -39,12 +39,9 @@ var (
 	// Default is 100%, not recommended for production use.
 	TraceSampling = env.RegisterFloatVar("PILOT_TRACE_SAMPLING", 100.0, "").Get()
 
-	// PushThrottle limits the qps of the actual push. Default is 10 pushes per second.
+	// PushThrottle limits the number of concurrent pushes allowed. Default is 100 concurrent pushes.
 	// On larger machines you can increase this to get faster push.
-	PushThrottle = env.RegisterIntVar("PILOT_PUSH_THROTTLE", 10, "").Get()
-
-	// PushBurst limits the burst of the actual push. Default is 100.
-	PushBurst = env.RegisterIntVar("PILOT_PUSH_BURST", 100, "").Get()
+	PushThrottle = env.RegisterIntVar("PILOT_PUSH_THROTTLE", 100, "").Get()
```

Comment:
do you still need this? the number of Istio users who can actually understand what this means and control this is about 1 or 2, while the number of users who have very large clusters is much more. I suggest removing this completely and sticking to the adaptive queuing that you had in your doc.

Reply:
I am not sure exactly what you mean by this. I think we need to have the number of pushes limited. Otherwise, we will get a push and try to spin up 1000+ goroutines (1 per proxy connected) and get throttled making all of the pushes slower.

The adaptiveness I mentioned is just that we are no longer setting a limit on pushes per second -- if pilot can push in 1ms we won't artificially limit it to only 100 pushes/s, but we will restrict the number of pushes it can trigger at one time